### DISCUSSION TITLE
[Shipped] New Performance Regression Issues for Endpoints and Functions

### DISCUSSION THREAD

**Discussion by nanoburd** [getsentry/sentry#61275] [Launch Feedback]
Hi I’m Albert, the Product Manager for Performance Monitoring and Alerting 👋 My main goal is to find strong signals in the performance data you send us so that you have clear and actionable steps to improve your application’s performance.

Today we’re launching a whole new category of Performance Issues called “Regression Issues” where we highlight large regressions in the time-series of your metrics. To differentiate this from regular “alerting”, we’re targeting two specific use cases to help jumpstart your troubleshooting with more context on where the root cause is.

# What are the new Issues?

Endpoint Regression issues and Function Regression issues both represent a more proactive way to address performance problems in your application before your customers start complaining.

## Endpoint Regression Issue

![Untitled (1)](https://github.com/getsentry/sentry/assets/97483437/50c2d50b-2c9b-42e5-b882-335a63dcb3d1)

Sentry monitors the p95 of your top **endpoints** in your backend projects and creates issues for sustained regressions. This is limited to specific transaction ops. See [[docs](https://docs.sentry.io/product/issues/issue-details/performance-issues/endpoint-regressions/)] for more info.

Issue details highlights suspect spans with a combination of frequency and duration to identify sources of the regression. Sentry also presents individual samples that represent the baseline vs regression for your inspection.

## Function Regression Issue

<img width="1062" alt="Untitled" src="https://github.com/getsentry/sentry/assets/97483437/4621e063-bed1-481c-b62e-969bcf82f525">

Sentry monitors the p95 of your most common **functions** in your project and creates issues for sustained regressions. You must be sending profiles to see this issue type. See [[docs](https://docs.sentry.io/product/issues/issue-details/performance-issues/function-regressions/)] for more info.

Issue details highlights the transactions this function is impacting across your application. Sentry also presents individual profiles that represent the baseline vs regression for your inspection.

## What’s Next

- **Web Vital Regression Issues** - Lets give our frontend developers some love so they can proactively address regressions in Web Vitals that impact their business and customer experience.
- **Promote Alerts into Issues** - This adds an option for you to configure your performance metric alerts to create an issue instead and allow you to provide additional context and data into the issue details to aid troubleshooting.

## We want to hear from you!

- What other metric regressions would you find useful?
- Would you want to tune the sensitivity of these?
- What other information would help you ‘solve’ these regressions faster?
- Do you care about these regressions as much as errors? In what scenarios is this as important as an error?

Feel free to share your thoughts in the discussion below!

**Comment by dgilmanAIDENTIFIED** [comment]
How do you turn this off or configure it? I can't see anything on it in the docs, and Settings -> <project> -> Performance has a laundry list of things you can tune but "Endpoint Regression" is not one of them.
